e2e/fixtures/fdb_cluster.go

```diff
@@ -119,6 +119,23 @@ func (fdbCluster *FdbCluster) Update() error {
 	return fdbCluster.getClient().Update(ctx.Background(), fdbCluster.cluster)
 }
 
+// UpdateNode update node definition
+func (fdbCluster *FdbCluster) UpdateNode(node *corev1.Node) error {
```

Comment:
This should be in the factory not in `fdb_cluster` as a node is a shared resource.